### ISSUE TITLE
#suggestion: ability to view who followed you so you can follow back

### ISSUE THREAD

**Issue by marrie** [streamlink/streamlink-twitch-gui#521]
I'm a visually impaired user of your software. I will be submitting some accessibility issues having to do with this app later. For now though I have a suggestion.

Since i have  trouble viewing who has followed me on twitch itself due to accessibility issues with their site I thought this program would be able to help. I cannot find a way to do this. The user should be able to do the following:

* view  who has followed them
* follow them back if at all possible
* elect to host them if they want to through the channels section of the \"my\" section.

This would be I think a good idea for those of us who don't want to go to the website  due to either accessibility issues or those of of us who want just an app for management etc. If this can be looked into that would be wonderful.

Running  Version 1.4.1 (1.4.1) of stream link twitch gui and the latest streamlink that was on homebrew as of 12-29-2017

Thanks.

**Comment by bastimeyer** [comment]
> I'm a visually impaired user of your software. I will be submitting some accessibility issues having to do with this app later.

Sure, but I don't know what I will be able to do other than tweaking/adding themes or improving some UI elements that would not be out of scope here.

> accessibility issues with their site I thought this program would be able to help
> who want just an app for management

This program is not meant to be a 1:1 replacement of the Twitch website. Its purpose is exclusively for browsing live streams that you can then watch via Streamlink. Anything that's related to broadcasting (hosting channels, seeing/managing your own followers, etc.) or which is generally not related to browsing/watching streams is not intended to be implemented in the future, sorry.
See #233...
